src/components/SuiteHeader/SuiteHeaderLogoutModal/SuiteHeaderLogoutModal.jsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 5737-M66, 5900-AAA
+ * (C) Copyright IBM Corp. 2020 All Rights Reserved.
+ * US Government Users Restricted Rights - Use, duplication, or disclosure
+ * restricted by GSA ADP Schedule Contract with IBM Corp.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Modal } from '../../../index';
+
+const defaultProps = {
+  isOpen: false,
+  i18n: {
+    heading: 'Do you wish to log out?',
+    secondaryButton: 'Cancel',
+    primaryButton: 'Log out',
+    body: (suiteName, userName) =>
+      `You are logged in to ${suiteName} as ${userName}.  Logging out also logs you out of each application that is open in the same browser.  To ensure a secure log out, close all open browser windows.`,
+  },
+};
+
+const propTypes = {
+  suiteName: PropTypes.string.isRequired,
+  displayName: PropTypes.string.isRequired,
+  isOpen: PropTypes.bool,
+  onClose: PropTypes.func.isRequired,
+  onLogout: PropTypes.func.isRequired,
+  i18n: PropTypes.shape({
+    heading: PropTypes.string,
+    primaryButton: PropTypes.string,
+    secondaryButton: PropTypes.string,
+    body: PropTypes.func,
+  }),
+};
+
+const SuiteHeaderLogoutModal = ({ suiteName, displayName, isOpen, onClose, onLogout, i18n }) => {
+  const mergedI18N = { ...defaultProps.i18n, ...i18n };
+  return (
+    <Modal
+      open={isOpen}
+      modalHeading={mergedI18N.heading}
+      primaryButtonText={mergedI18N.primaryButton}
+      secondaryButtonText={mergedI18N.secondaryButton}
+      onSecondarySubmit={() => onClose()}
```

Comment:
can we not just pass the function reference here instead of wrapping in arrow function?
```suggestion
      onSecondarySubmit={onClose}
```

Reply:
Because `propTypes` don't allow you to define an expected function signature, I typically call any prop functions explicitly so that the component consumer can see in the source exactly what parameter signature is expected. 
 `onSecondarySubmit={onClose}` passes the click event along when it's not expected.  Happy to change it to be consistent with other PAL usage though, just let me know!

Reply:
(we discussed offline, I agree we should minimize the number of anonymous functions that aren't wrapped in `useCallback` -- i'll fix these)